plasmapy/particles/isotopes.py

```diff
@@ -11,7 +11,7 @@
 import json
 import pkgutil
 
-# this code was used to create the JSON file as per vn-ki on Riot:
+# this code was used to create the JSON file as per vn-ki on Matrix:
 # https://matrix.to/#/!hkWCiyhQyxiYJlUtKF:matrix.org/
```

Comment:
I left this as matrix.to, since it's in the code and we can assume some level of previous knowledge.